mongo/datadog_checks/mongo/collectors/replication_info.py

```diff
@@ -0,0 +1,51 @@
+import pymongo
+
+from datadog_checks.base.utils.common import round_value
+from datadog_checks.mongo.collectors.base import MongoCollector
+
+
+class ReplicationInfoCollector(MongoCollector):
```

Comment:
Given that these metrics only relate to the operation log, should we rename the class to something more specific? Like:

```suggestion
class ReplicationOpLogCollector(MongoCollector):
```